docs/how_to/use_toggles.rst

```diff
@@ -0,0 +1,65 @@
+***************************************
+How to: Use toggles
+***************************************
+
+Calling toggles
+==================
+
+Unlike the underlying waffle library which looks up waffles by name,
+toggles should be imported and used directly:
+
+.. code:: python
+
+    from whereever import FLAG_FOR_STUFF, COURSE_FLAG_FOR_STUFF
+
+    FLAG_FOR_STUFF.is_enabled()
+    COURSE_FLAG_FOR_STUFF.is_enabled('course-key-123')
+
+You can get at the state of these toggles by name via the waffle library
+and it will appear to work fine in simple testing. Don't be fooled.
+
+How toggles get their results
```

Comment:
Again, where do these docs best belong? This doesn't feel right here, but might need more discussion.